model/api_endpoint_lb.go

```diff
@@ -83,11 +83,20 @@ func (e APIEndpointLB) ManageSecurityGroup() bool {
 	return !e.NetworkLoadBalancer() && len(e.APIAccessAllowedSourceCIDRs) > 0
 }
 
+// ManageTargetGroup returns true if kube-aws should create a target group for this ELB
```

Comment:
As discussed, please take a look at this - it sounded like you might not need it! :+1:

Reply:
done!